wordle-web/src/main.ts

```diff
@@ -20,6 +20,25 @@ const vuetify = createVuetify({
     sets: {
       mdi
     }
+  },
+  theme: {
+    defaultTheme: "dark",
+    themes: {
+      dark: {
+        colors: {
+          correct: '#4CAF50',
+          misplaced: '#FFC107',
+          wrong: '#F44336'
+        }
+      },
+      inverse: {
+        colors: {
+          correct: '#F44336',
+          misplaced: '#03A9F4',
```

Comment:
I don't like this color
```suggestion
          misplaced: '#FFC107',
```